### DISCUSSION TITLE
Navbar Improvements

### DISCUSSION THREAD

**Discussion by deo002** [eclipse-sw360/sw360-frontend#163] [General]
Navbar component has become overcomplicated. We need to refactor and decompose it to make it proper.

**Comment by hoangnt2** [comment]
I think we can add **nested** property to navbar items like this. If the item has **nested** properties, It will show the dropdown list. Then we will refactor the Navbar component to fit the new format.

```
[
    {
        "link": "/home",
        "name": "Home"
    },
    {
        "link": "/vulnerabilities",
        "name": "Vulnerabilities"
    },
    {
        "link": "/admin",
        "name": "Admin"
        "nested": [
            {
                "link": "/obligations",
                "name": "Obligations"
            },
            {
                "link": "/oauth-client",
                "name": "OAuth Client"
            }
        ]
    },
    {
        "link": "/preferences",
        "name": "Preferences"
    }
]
```

**Comment by heliocastro** [comment]
@deo002 @hoangnt2 I think this discussion over, no ? 
Navbar for the current state is stable and we can expect changes only on the "After19" UI framework change plans.
